app/models/concerns/labware_creators/require_wells_with_sample_metadata.rb

```diff
@@ -1,28 +1,28 @@
 # frozen_string_literal: true
 
-# Can be included in plate creators which requires well aliquots to have a sample manifest
-module LabwareCreators::RequireWellsWithSampleManifest
+# Can be included in plate creators which requires well aliquots to have sample metadata
+module LabwareCreators::RequireWellsWithSampleMetadata
```

Comment:
I think it would be more informative to call it `RequireWellsWithCollectedBy` (and equivalent changes to the method name etc.), as the current name implies it could be used by anyone who just wanted sample metadata, not that specific field.